docs/TRANSCRIPTION.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Build a Real-Time Transcription Backend with Astro
+description: Learn how to build a backend to initiate Real Time Transcription using Astro.
+---
+
+50% of Americans watch content with subtitles. Why can't we have this with video calls? Actually, we can do this by using Real-Time Transcription with Agora.
+
+This guide is the third part in a series about building backends with Astro. The first guide covered Token Generation with Astro so we can secure our video calls. We will use the token generation logic in this guide. The second guide was about recording your video to a storage provider of your choice. Real-Time Transcription is a separate feature, so it does not rely on the cloud recording guide, but we will need to generate tokens. Let's get started.
+
+## Prerequisites
+1. NodeJS and Astro installed.
+2. A developer account with [Agora](https://console.agora.io/).
+3. A token generator built with Astro. Find the [guide here](https://github.com/AgoraIO-Community/astro-backend/blob/main/docs/TOKENS.md).
+
+## Project Setup
+This guide will build upon the [token generator guide](https://github.com/AgoraIO-Community/astro-backend/blob/main/docs/TOKENS.md). The token generator guide walks through building the `api/tokens.json` endpoint, which returns a token used to secure your video calls. This endpoint uses a `handleGenerateToken` function that handles token generation logic. Since it is on the same backend server, we will use this function directly within the real-time transcription endpoint.
+
+From the previous guide, you should have `APP_ID` and the `APP_CERTIFICATE` in your environment variables. We will use the `APP_ID` to start and stop the transcription.
+
+## Real Time Transcription Overview
```

Comment:
This section needs to be much earlier in the guide. There is a jump between the intro and the pre-requisites / project setup, as a reader it feels like I'm being dropped into a situation but there's some context im missing so it's not clear what's going on until i get to this section. I think moving this up in the guide's hierarchy will resolve that confusion.

Reply:
Hmmmm. This is following the same format as the Token and Cloud Recording Guide.

1. Intro 
2. Prerequisites
3. Setup
4. Overview